content/09.evolution.md

```diff
@@ -83,7 +83,32 @@ This pattern of convergent evolution at some sites could indicate that certain m
 While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing and future evolution, the relationship between mutations and pathogenicity is not yet known.
 Additional data is needed in order to understand patterns of evolutionary change and whether they are likely to affect virulence.
 
-<!-- TO DO: Add info on UK and South African variants.-->
+Several factors could promote the evolution of SARS-CoV-2 including immunodeficiency and the transient exposure to antibodies directed against SARS-CoV-2 proteins. 
+A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia (CLL) and hypogammaglobulinemia [@doi:10.1016/j.cell.2020.10.049] suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency. 
```

Comment:
```suggestion
A single case study of SARS-CoV-2 infection in an immunocompromised female with chronic lymphocytic leukemia and hypogammaglobulinemia [@doi:10.1016/j.cell.2020.10.049] suggested that an accelerated evolution of the virus could occur in conditions of immunodeficiency. 
```
I don't think CLL is used 3+ times so mSystems doesn't require that it be defined!

Reply:
makes sense!